### ISSUE TITLE
Combining textInput with enable

### ISSUE THREAD

**Issue by smoelker** [ErikSchierboom/knockout-pre-rendered#56]
I'm having trouble combining the 'textInput' and 'enable' bindings. I isolated the issue here on jsfiddle: [https://jsfiddle.net/xazynusL/13](https://jsfiddle.net/xazynusL/13)

In this example I expect the value of the textbox to be 'test' and not 'true'. Removing the 'enable' binding seems to work but that's not a solution for me.

Am I doing something wrong here or is this a bug?

**Comment by revengineering** [comment]
I believe the `init` binding when used without parameters is intended as shorthand that may only work properly when combined with _one other binding_. (e.g. `data-bind="init, textInput: text"`).  If you have multiple bindings, then the `init` shorthand may not work predictably.

Instead, for your case, you can do the following:
```
<input type="text" data-bind="init: { text: 'test' }, textInput: text, enable: enabled" value="test" />

or even omit the value attribute:

<input type="text" data-bind="init: { text: 'test' }, textInput: text, enable: enabled" />
```

**Comment by smoelker** [comment]
Your solution works but is not very satisfying to me. The purpose of this library is to integrate pre-rendered html with knockout bindings smoothly.  Having to set the binding value 'manually' this way seems to not contribute to the purpose.

The reason why it's extra painful for me is because I use Symfony Forms which renders the html-form for me. I do have the ability to override attributes and, therefore, can customize the data-bind attribute but it will get a bit messy. Example:
```html
<div class="form-group">
{{ form_widget(form.text, {
    'attr': {
        'data-bind': 'init: {text: "' ~ form.text.vars.value  ~'"}, textInput: text, enable: enabled'
    }
}) }}
</div>
```

While I would prefer just:

```html
<div class="form-group">
{{ form_widget(form.text, {
    'attr': {
        'data-bind': 'init, textInput: text, enable: enabled'
    }
}) }}
</div>
```

I haven't looked into the source code of the project but do you know of there is a good reason why this works the way it does now?

**Comment by revengineering** [comment]
That would be a question for Erik. I've looked at the code, and I understand _why_ it behaves as you see, given how the code is currently.  Changing it so that it can initialize _all the bindings_ might be possible. Not sure of the effort.

**Comment by revengineering** [comment]
Ok, so I've got a local working prototype that initializes _all_ bindings successfully. 

Something like the following:
```
<a  data-bind="init, attr: { href: link }, text: name, enable: enabled" 
    href="https://www.google.com" 
    disabled>Google</a>
```
would populate the model such that:
```
model.link() == "https://www.google.com"
model.name() == "Google"
model.enabled() == false
```
Still have some cleanup to do. The changes are more verbose than I wanted them to be.

**Comment by ErikSchierboom** [comment]
> I haven't looked into the source code of the project but do you know of there is a good reason why this works the way it does now?

@smoelker No particular reason, the idea was to just keep it as simple as possible.

@revengineering Great! Looking forward to the PR.

**Comment by revengineering** [comment]
Slow progress on this. I've been a bit overwhelmed with other projects at the moment. But I haven't forgotten about it, and will actually need it soon enough, I think.

**Comment by revengineering** [comment]
Finally got back around to this. PR coming later today or tomorrow, with slight refactoring of the init logic to support multiple bindings. Also adds support for the css binding.

**Comment by ErikSchierboom** [comment]
Okay nice!

**Comment by ErikSchierboom** [comment]
I've just released version 0.11.0, which includes the fix for this issue. Thanks a lot for doing this @revengineering!
